art-bot.py

```diff
@@ -275,14 +277,24 @@ def respond(client: RTMClient, event: dict):
                 'regex': r'^.*(image )?pipeline \s*for \s*image \s*(registry.redhat.io/)*(openshift4/)*(?P<delivery_repo_name>[a-zA-Z0-9-]+)\s*( \s*in \s*(?P<version>\d+.\d+))?\s*$',
                 'flag': re.I,
                 'function': pipeline_from_delivery
-            }
+            },
 
+            # Others
+            {
+                'regex': r'^Alert if https://(?P<release_browser>[\w]+).ocp.releases.ci.openshift.org(?P<release_url>[\w/.-]+) (stops being blue|fails|is rejected|is red|is accepted|is green)$',
```

Comment:
I'd probably make this so I don't have to remember:
```suggestion
                'regex': r'^Alert ?(if|when|on)? https://(?P<release_browser>[\w]+).ocp.releases.ci.openshift.org(?P<release_url>[\w/.-]+) ?(stops being blue|fails|is rejected|is red|is accepted|is green)?$',
```
:shrug:

Reply:
Done!